readme.md

```diff
@@ -93,7 +118,10 @@ Your Answer:
 How is the concept of OAuth related to a valet key?
 
 Your Answer:
-```text
+```
+OAuth is a security protocol that allows user to grant third-party access to their web resources without sharing their passwords.
+
+A valet key gives access to certain things while using another key to unlock everything else...?
 
```

Comment:
Valet keys allow you to give someone the keys to your car without full access (e.g. they can't open the trunk or glovebox). Similarly, OAuth allows you to share your account on a 3rd party provider (e.g. Facebook, Twitter, Google), without given the consumer app your password, or necessarily full access to your account, such as reading / modifying  your data, posting on your behalf, etc.